bedrock/mozorg/templates/mozorg/credits-faq.html

```diff
@@ -51,7 +51,7 @@ <h2>Why do I need to include a citation?</h2>
 
 <h2>When will I get added, and how do I know?</h2>
 
-<p>Additions are normally done in batches, a few months apart. You will receive an email when your name is added, or requesting more information if there has been a problem with your application (e.g. additional verification is needed). When you have been added, you will see your name and citation in <a href="https://github.com/mozilla/community-data/commits/master/credits/names.csv">the log</a>.</p>
+<p>Additions are normally done in batches, a few months apart, you will receive an email when your name is added. When you have been added, you will see your name and citation in <a href="https://github.com/mozilla/community-data/commits/master/credits/names.csv">the log</a>. Your name will not be added if your application does not meet the criteria mentioned above, in that case feel free to re-apply once you have a stronger citation and evidences.</p>
```

Comment:
Nit: I'm not sure the changes here read quite right as sentances in their own merit. It makes sense overall, but I'd suggest revising the punctuation slightly.

Reply:
How about following text? If this looks good to you, I will update the PR. (also notice I updated word log to logs below)..

Additions are normally done in batches, usually a few months apart. You will receive an email when your name is added and you will also see your name and citation in <a href="https://github.com/mozilla/community-data/commits/master/credits/names.csv">the logs</a>. If your application does not meet the criteria mentioned above, your name will not be added and in that case re-apply once you have a stronger citation and evidences.

Reply:
First sentance reads nice. For the second, how about:

```
If your application does not meet the criteria mentioned above, your name will not be added. You are always welcome to re-apply once you have a stronger citation and further evidence.
```